app/controllers/ValidationController.scala

```diff
@@ -30,6 +32,20 @@ class ValidationController @Inject() (implicit val env: Environment[User, Sessio
   val validationMissionStr: String = "validation"
   val mobileValidationMissionStr: String = "validation"
   val rapidValidationMissionStr: String = "rapidValidation"
+  val mobileOS: Regex = "(iPhone|webOS|iPod|Android|BlackBerry|mobile|SAMSUNG|IEMobile|OperaMobi)".r
```

Comment:
Where did you find this particular list for the regex? I feel like this probably is not comprehensive. This stackoverflow post seems to suggest using a more comprehensive regex: https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser

And I know we've been saying mobile, but we want to include tablets with mobile devices. Our website doesn't support touchscreens well.